### PR TITLE
7.17

### PR DESCRIPTION
**Pull request by mhugo**
[elasticsearch-plugin-geoshape-7.17.0.0.zip](https://github.com/opendatasoft/elasticsearch-plugin-geoshape/files/8080388/elasticsearch-plugin-geoshape-7.17.0.0.zip)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -236,7 +240,10 @@ Note that the duplicated point has been deduplicated.
 
 ### Geoshape aggregation
 
-Geoshape aggregation based on auto-computed shape hash.
+This aggregation creates a buket for each input shape (based on the hash of its WKB representation) and compute a simplified version of the shape in the bucket.
```
Comment (benj-moreau) [inline README.md]:
```suggestion
This aggregation creates a bucket for each input shape (based on the hash of its WKB representation) and compute a simplified version of the shape in the bucket.
```

**Review by benj-moreau — APPROVED** [review]
Plugin is building and working well

**Comment by 5k4nd** [comment]
After some investigation, we cannot easily drop the `legacygeo` dependency for now because there is no ES-native replacement for WKB encoder.

We'll have to wait until ES 8.1, which includes a new `WellKnownBinary` encoder ([here](https://github.com/iverase/elasticsearch/blob/79b772867450cefb7d05fbaab428ec8820952067/libs/geo/src/main/java/org/elasticsearch/geometry/utils/WellKnownBinary.java)) thanks to [this PR](https://github.com/elastic/elasticsearch/pull/82706).

**Review by 5k4nd — APPROVED** [review]

**Comment by mhugo** [comment]
Thanks for the investigation !

**Pull request merged**